bin/node-template/runtime/src/lib.rs

```diff
@@ -500,6 +505,7 @@ impl_runtime_apis! {
 			let mut batches = Vec::<BenchmarkBatch>::new();
 			let params = (&config, &whitelist);
 
+			add_benchmark!(params, batches, frame_benchmarking, BaselineBench::<Runtime>);
```

Comment:
is the idea with adding it to the node-template that we want new substrate users to get into the habit of using this baseline benchmark pallet in their project?

Reply:
sure, if we use it, they should too. And it is pretty free with no impact.